<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="fileTextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get text file</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="plainTextBrowser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>851</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_genPrivateKey">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Keys</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_PrivatKey">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>851</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MD5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MD5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>no Init</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_PublicKey">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>861</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Code">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CodingStr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>String for RSA coding:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_forEncrypting">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypted:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Encrypted">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>570</y>
      <width>731</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypted:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Decrypted">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>660</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
